resources/leiningen/new/kraken/config.edn

```diff
@@ -1,3 +1,7 @@
 {:rabbit-mq {:connection {:host #resource-config/env "RABBITMQ_PORT_5672_TCP_ADDR"
                           :port #resource-config/edn #resource-config/env "RABBITMQ_PORT_5672_TCP_PORT"}
-             :queue "{{name}}"}}
+             :queues {"{{name}}.ok" {:exclusive false :auto-delete true}}}
+ :datomic {:uri "datomic:mem://{{name}}"
+           :partition :{{name}}
+           :intitialize true
```

Comment:
Feel free to declare this question out of scope, but is there really any need to ever not initialize datomic? Doesn't creating the DB just return false (and leave the data intact) when it's already there?

I think we were wanting to make sure that we didn't silently create a database if the production one was unexpectedly missing. But if the production database is missing, we've got bigger problems.